src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -780,6 +785,10 @@ private function createProviderFromDSN(DSN $dsn): Document
                     'messageId' => $dsn->getParam('messageId'),
                     'useDLT' => $dsn->getParam('useDLT'),
                 ],
+                'inforu' => [
+                    'senderId' => $dsn->getParam('senderId', $from),
```

Comment:
```suggestion
                    'senderId' => $user,
```
we can just pass the senderID as the user in the DSN right? 

`sms://senderId:password@inforu`

Reply:
yeah, updated